Clarify "Areas of focus" in `People/Global_Moderation_Team` (#11579)
ppy/osu-wiki: wiki/People/Global_Moderation_Team/en.md

The [Global Moderation Team group page](https://osu.ppy.sh/groups/4) lists all of the team members. In addition to areas mentioned below, all GMT members take part in [reviewing visual and aural content included in beatmaps](/wiki/Rules/Visual_content_considerations#getting-your-image-assessed).

> **A member of the GMT can engage with any tasks within any category that they choose, despite their listed primary responsibilities.** For example, a GMT member primarily responsible for chat moderation may also help with forum moderation, or vice versa.

Reviewer: ```suggestion
**A member of the GMT may engage with any task within any category that they choose, regardless of their listed area of focus.** For example, a GMT member primarily responsible for chat moderation may also help with forum moderation.
```

Author: Same wording used as in the [NAT Article](https://osu.ppy.sh/wiki/en/People/Nomination_Assessment_Team#team-members:~:text=A%20member%20of%20the%20NAT%20can%20engage%20with%20any%20tasks%20within%20any%20category%20that%20they%20choose%2C%20despite%20their%20listed%20primary%20responsibilities.%20For%20example%2C%20an%20NAT%20member%20primarily%20responsible%20for%20evaluations%20may%20also%20help%20with%20structural%2Drelated%20tasks%2C%20or%20vice%20versa.). If one is slightly modified, may as well do the other one.

Reviewer 2: can be done in this same pr

Author: Will do then

<!-- NOTE FOR TRANSLATORS: Translating this section is different from other parts of the osu! wiki. See https://github.com/ppy/osu-wiki/blob/master/meta/group-info/TRANSLATING.md#group-articles. -->

> | Name | Additional languages | Area of focus |

Reviewer: Petal missing from this table

Author: Asking for their area of focus, then adding them.

Author: They wanted it empty. Added as such.

| :-- | :-- | :-- |
| ::{ flag=MY }:: [\_Kobii](https://osu.ppy.sh/users/6209713) | Chinese, Malay, Cantonese, some Japanese | Mapping/modding community moderation |
| ::{ flag=DE }:: [- Felix](https://osu.ppy.sh/users/8503985) | German | Chat moderation |
| ::{ flag=CH }:: [\[ryuu\]](https://osu.ppy.sh/users/5698467) | Russian | Chat moderation |
| ::{ flag=RS }:: [0x84f](https://osu.ppy.sh/users/7944724) | Serbian |  |
| ::{ flag=US }:: [abraker](https://osu.ppy.sh/users/4635891) |  | Forum moderation |
| ::{ flag=NL }:: [Albionthegreat](https://osu.ppy.sh/users/9853595) | Dutch, some German |  |
| ::{ flag=CA }:: [Azer](https://osu.ppy.sh/users/2155578) |  | Tournament management[^task-Azer] |
| ::{ flag=MY }:: [bibitaru](https://osu.ppy.sh/users/4482419) | Chinese, Malay | Chat moderation |

> | ::{ flag=US }:: [Blushing](https://osu.ppy.sh/users/5927823) |  | Chat moderation, forum moderation |

Reviewer: do they moderate `#greek` (for table below) ? jw cuz I saw them contribute something to greek on the wiki before.

Author: Was not given any additional languages when I asked, so we assume not.
